.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
```

Comment:
Uh, not going to work when deps are in `build.gradle`.

AFAIK use of Dependabot on Gradle-based Jenkins plugins is untested and may or may not work sanely at all.